src/features/snackbar/index.js

```diff
@@ -107,6 +117,7 @@ class Snackbar extends Component {
             this.setState({
                 show: `SOLD ITEM: ${itemSold.split('-')[0]}`,
                 item: item,
+		equip: false,
```

Comment:
Is there a reason this line has a different level of indentation?

Reply:
Ah, that's probably because I wrote it on my phone which doesn't have `eslint` installed and thus doesn't correct the formatting on commit. I'll run the linter and fix it.